angular/projects/admin-nrpti/src/app/explore-panel/explore-panel.component.html

```diff
@@ -158,7 +159,11 @@
           <option [selected]="selectedSystemRef === 'ocers-csv'" value="ocers-csv">OCERS</option>
           <option [selected]="selectedSystemRef === 'nris-epd'" value="nris-epd">NRIS</option>
           <option [selected]="selectedSystemRef === 'epic'" value="epic">EPIC</option>
-        </select>
+        </select> -->
+        <mat-label>Source System</mat-label>
```

Comment:
CSS styling is different from the other UI components on the screen, looks like it's just because it's a select vs. a auto-complete select like ActivityType, etc. Did we want this component to work the same as the other selectors in the filter panel?

Reply:
The select has different functionality as auto-complete. At the moment app-autocomplete-multi-select has a few bugs as well. It is likely we will re-write it. During that time, we can unify the styling between select and auto-complete.